### ISSUE TITLE
Unknown key for a START_OBJECT in [data]

### ISSUE THREAD

**Issue by n1k0** [Kinto/kinto-elasticsearch#17]
When searching by sending an empty POST request to search:

```
$ http POST http://0.0.0.0:8888/v1/buckets/default/collections/toto/search -a toto:toto
```

Response

```
HTTP/1.1 200 OK
Access-Control-Expose-Headers: Alert, Backoff, Content-Length, Retry-After
Content-Length: 2
Content-Type: application/json
Date: Tue, 23 May 2017 13:43:10 GMT
Server: waitress
X-Content-Type-Options: nosniff

{}
```

Error in the server logs:

```
elasticsearch.exceptions.RequestError: TransportError(400, 'parsing_exception', 'Unknown key for a START_OBJECT in [data].')
```

**Comment by leplatrem** [comment]
I cannot reproduce :(

**Comment by leplatrem** [comment]
Partly fixed with #26
